### ISSUE TITLE
dynamic focusable element makes problems on navigation

### ISSUE THREAD

**Issue by nicecue** [dead/react-key-navigation#3]
Hello, I'm working with your library.
I found some problems on dynamic focusable element insertion.
If focusable elements is inserted dynamically, focus order doesn't work correctly.
Did you have any chance to meet this problem before??

**Comment by dead** [comment]
Can you give me a simple example in https://codesandbox.io ?
Thanks!

**Comment by edwardmhunton** [comment]
Hi there Guys
I think you might have the same issue as me...I set a Focusable component as 'active' but the navigation uses indexOfParent s its index and this is always 0...

Did you find a solution for this? 

Otherwise, a great library

**Comment by nicecue** [comment]
Hi there,
sorry for late answer, I was busy

check this out  ===> [example](https://codesandbox.io/s/qqmv743yk4)

**Comment by dead** [comment]
Hey guys!
Thanks for the example!
I found the problem, it happens because the focusable component is inserted at the end of the parent children list...

This isn't the best way but you can avoid this issue by doing this:
```jsx
<VerticalList>
{this.state.dynamicInsert && <Search text="inserted!!" />}
</VerticalList>
```

I don't know if is possible to know the order of the children components so I'm accepting suggestions in how to fix this problem...

**Comment by pahonski** [comment]
Hello! Did you find the solution for this problem with dynamic elements?
